base/stats.go

```diff
@@ -421,8 +421,8 @@ type CacheStats struct {
 	NonMobileIgnoredCount *SgwIntStat `json:"non_mobile_ignored_count"`
 	// The total number of active channels.
 	NumActiveChannels *SgwIntStat `json:"num_active_channels"`
-	// The total number of skipped sequences.
-	NumSkippedSeqs *SgwIntStat `json:"num_skipped_seqs"`
+	// The total number of skipped sequences. This is a cumulative value.
+	NumSkippedSeqs *SgwIntStat `json:"cumulative_num_skipped_seqs"`
```

Comment:
We want to avoid renaming existing stats, so we don't break anything users have built on the existing names.